{"name": "Проверка допустимого количества символов в полях ввода", "status": "failed", "statusDetails": {"message": "AssertionError: No message", "trace": "browser = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"afa216052ac26a6ac50982fba756ffe3\")>, base_url = 'http://172.19.16.142/', first_name = 'pnEXOFSmN1olAkQHeGxgB2DqsrJ8UTf0', second_name = 'w', password = '6'\n\n    @allure.title(\"Проверка допустимого количества символов в полях ввода\")\n    @allure.description(\"Проверка граничных условий полей для ввода имени, фамилии и пароля\")\n    @pytest.mark.parametrize(\"first_name\",[UserLoginPage.generate_string(num_of_chars=1),\n                                           UserLoginPage.generate_string(num_of_chars=32, num=True, uppercase=True)])\n    @pytest.mark.parametrize(\"second_name\",[UserLoginPage.generate_string(num_of_chars=1),\n                                           UserLoginPage.generate_string(num_of_chars=32, num=True, uppercase=True)])\n    @pytest.mark.parametrize(\"password\",\n                             [UserLoginPage.generate_string(num_of_chars=1, num=True, uppercase=True, sym=True),\n                              UserLoginPage.generate_string(num_of_chars=32, num=True, uppercase=True, sym=True)])\n    def test_border_conditions(browser, base_url, first_name, second_name, password):\n        new_user = UserLoginPage(browser)\n        new_user.open_reg_account_page(base_url)\n        new_user.fill_in_with_letters(new_user.FIRST_NAME_FIELD, name=first_name)\n        new_user.fill_in_with_letters(new_user.LAST_NAME_FIELD, name=second_name)\n        new_user.enter_email()\n        new_user.enter_telephone()\n        passwd = new_user.enter_password(pwd=password)\n        new_user.fill_in_with_letters(new_user.PASSWORD_CONFIRM_FIELD, name=passwd)\n        new_user.find_web_element(new_user.PRIVACY_POLICY_CHECKBOX).click()\n        new_user.wait_and_click(new_user.SUBMIT_CONTINUE_BUTTON)\n        if new_user.view_success_notification() != \"Your Account Has Been Created!\":\n            browser.save_screenshot(new_user.screenshot_name)\n            new_user.logger.error(\"Account creation message was not displayed \")\n>           raise AssertionError(\"No message\")\nE           AssertionError: No message\n\ntest_reg_account.py:66: AssertionError"}, "description": "Проверка граничных условий полей для ввода имени, фамилии и пароля", "attachments": [{"name": "log", "source": "502079a4-25f2-4c52-bfc8-588b1152bc04-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "password", "value": "'6'"}, {"name": "second_name", "value": "'w'"}, {"name": "first_name", "value": "'pnEXOFSmN1olAkQHeGxgB2DqsrJ8UTf0'"}], "start": 1640005601309, "stop": 1640005605485, "uuid": "9dcc2f72-885a-4883-807f-dd5c817425e6", "historyId": "4daac187271be95ec1e8f91ff3adf4d8", "testCaseId": "bb2220456bb940b18fcd35ce6dab0375", "fullName": "test_reg_account#test_border_conditions", "labels": [{"name": "suite", "value": "test_reg_account"}, {"name": "host", "value": "sis002422"}, {"name": "thread", "value": "13004-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_reg_account"}]}